benchmarks/nel/project.yml

```
@@ -89,9 +86,9 @@ commands:
       - "temp/${vars.dataset}/nlp"
 
   - name: compile_corpora
-    help: "Compile corpora, separated in in train/dev/test sets"
+    help: "Compile corpora, separated in in train/dev/test sets."
     script:
-      - "python ./scripts/compile_corpora.py ${vars.dataset}"
+      - "python ./scripts/compile_corpora.py ${vars.dataset} True"
```

Since you're using Typer can't you use a flag here?

```suggestion
      - "python ./scripts/compile_corpora.py ${vars.dataset} --use-filter-terms"
```

---

Correct, replaced that meanwhile.